.gitignore

```diff
@@ -1,7 +1,11 @@
+.vscode
+**/__pycache__
```

Comment:
they are slightly different [apparently](https://git-scm.com/docs/gitignore): `*` matches everything except a '/' and `**` matches everything including a '/', so we need to use the later for deeply nested `__pycache__`s

Reply:
And I chose `**` over `*/*/` simply because I wanted to exclude them from the whole project in case I decided to move the py scripts or we add more py scripts in other locations in the future

Reply:
Oh cool, I didn't realize that!